### ISSUE TITLE
Dragging causes a framerate dip

### ISSUE THREAD

**Issue by adam-savard** [Mwni/pixi-text-input#42]
Just flagging this as an issue for anyone else that runs across it!

The framerate will take a massive nosedive if you have dozens of text inputs on screen and you're dragging them inside a container; this is because of the bounds calculation that happens in the `_update`. There's probably a more efficient way to make this happen, though I don't have any suggestions at this time.

**Comment by Mwni** [comment]
Good point. This could be optimized by not updating the HTML text input internally when the PIXI surrogate is being drawn instead. Of course, if `substituteText = false`, then this does not apply.
